docs/GroupDefs.hpp

```diff
@@ -833,6 +833,17 @@ then typically look as follows:
  * inside the debugger.
  */
 
+/*!
+ * \defgroup CoordMapsTimeDependentGroup  Time-dependent Coordinate Maps
```

Comment:
Coordinate Maps, time-dependent   (you will need to move it next to CoordinateMaps in the file)